test/jdk/java/io/IO/IO.java

```diff
@@ -55,12 +55,14 @@ public class IO {
     public class OSSpecificTests {
 
         private static Path expect;
+        static final int JCK_STATUS_BASE = 95;
 
         @BeforeAll
         public static void prepareTTY() {
             expect = Paths.get("/usr/bin/expect"); // os-specific path
             if (!Files.exists(expect) || !Files.isExecutable(expect)) {
-                throw new SkippedException("'" + expect + "' not found");
+                System.out.println("jtreg.SkippedException: '" + expect + "' not found");
```

Comment:
SkippedException works with jtreg tests only. For jUnit you need to use [Assumptions.abort](https://junit.org/junit5/docs/5.9.1/api/org.junit.jupiter.api/org/junit/jupiter/api/Assumptions.html#abort(java.lang.String))

Reply:
> SkippedException works with jtreg tests only. For jUnit you need to use [Assumptions.abort](https://junit.org/junit5/docs/5.9.1/api/org.junit.jupiter.api/org/junit/jupiter/api/Assumptions.html#abort(java.lang.String))

Yes, the Assumptions API should be used here. We use that in several JUnit tests that skip when tests when they can't run and you want it to fail the test.

Reply:
Thanks for the review and suggest. The code has been updated according the suggest. Thanks.